projects/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
We don't check in IDE configs. Please remove this file.

Reply:
okay, file is removed in latest commit.